### PR TITLE
Update Prepend Regex

### PR DESCRIPTION
**Pull request by apaniukov**
CVS-156583

### CODE REVIEW

File: src/regex_normalization.cpp [hunk 1]
```diff
@@ -18,20 +17,36 @@ namespace {
  * @return std::string Reformatted replace pattern
  */
 std::string reformat_replace_pattern(std::string replace_pattern) {
-    return std::regex_replace(replace_pattern, std::regex(R"((?:\\)([0-9]+))"), R"($$1)");
+    for (char i = '1'; i <= '9'; ++i) {
```
Comment (pavel-esir) [inline src/regex_normalization.cpp:20]: Why did you change this implementation?

**Review by pavel-esir — APPROVED** [review]

File: src/regex_normalization.cpp (hunk 1 above)
Comment (apaniukov) [reply src/regex_normalization.cpp:20]: Regex didn't work in some cases.
